.pre-commit-config.yaml

```diff
@@ -1,17 +1,18 @@
+ci:
+    autofix_commit_msg: |
+      [pre-commit.ci] auto code formatting
+    autofix_prs: true
+    autoupdate_branch: ''
+    autoupdate_commit_msg: '[pre-commit.ci] pre-commit autoupdate'
+    autoupdate_schedule: quarterly
+    skip: []
+    submodules: false
+
 repos:
       - repo: https://github.com/pre-commit/mirrors-clang-format
         rev: v11.1.0
         hooks:
               - id: clang-format
-                # Using the pre-commit stage to simplify invocation of all
-                # other hooks simultaneously (via any other hook stage).  This
-                # can be removed if we also move to running clang-format
-                # entirely through pre-commit.
-                stages: [commit]
-                name: clang-format
-                description: Format files with ClangFormat.
-                entry: clang-format
-                language: system
                 files: \.(cu|cuh|h|hpp|cpp|inl)$
                 'types_or': [file]
```

Comment:
Is there a reason to manually specify the supported file extensions rather than use the type identifiers?

Reply:
Good question. The type identifier can identify all extensions but `.inl`:
```bash
(base) yunsongw@yunsongw-dt:~/Work/cuCollections/include/cuco/detail$ identify-cli static_map.inl 
["file", "non-executable", "text"]
```
Maybe I should create an issue at `pre-commit` repo to add `.inl` as `header` or `c++`.

Reply:
I think you'd want inl to be considered C++.

Reply:
Done in https://github.com/pre-commit/identify/pull/273